components/aws/sagemaker/tests/integration_tests/conftest.py

```diff
@@ -180,3 +182,76 @@ def experiment_id(kfp_client, tmp_path_factory, worker_id):
             data = get_experiment_id(kfp_client)
             fn.write_text(data)
     return data
+
+
+@pytest.fixture(scope="session")
+def deploy_endpoint(sagemaker_client, s3_data_bucket, sagemaker_role_arn, region):
+    model_name = utils.generate_random_string(5) + "-model"
```

Comment:
Can you add something in the prefix/suffix that makes it clear that this is part of the model monitoring test.